tests/python_package_test/test_dask.py

```diff
@@ -118,6 +257,27 @@ def test_training_does_not_fail_on_port_conflicts(client):
             )
             assert dask_classifier.booster_
 
+    client.close()
+
+
+@pytest.mark.parametrize('output', data_output)
+@pytest.mark.parametrize('centers', data_centers)
+def test_classifier_proba(output, centers, client, listen_port):
```

Comment:
I think this test might have come from a mistake resolving a merge conflict. I merged this into the `test_classifier` test in #3786 . Could you please remove it?

Reply:
oops my bad!

Reply:
Addressed in d1d4abf